third_party/maya/lib/usdMaya/shadingModeExporter.h

```diff
@@ -46,7 +46,8 @@ class PxrUsdMayaShadingModeExporter {
     virtual void DoExport(const UsdStageRefPtr& stage,
                           const PxrUsdMayaUtil::ShapeSet& bindableRoots,
                           bool mergeTransformAndShape,
-                          const SdfPath& overrideRootPath);
+                          const SdfPath& overrideRootPath,
+                          PxrUsdMayaUtil::MDagPathMap<SdfPath>::Type& dagPathToUsdMap);
```

Comment:
Do we expect shading exporters to introduce new entries in the dagPathToUsdMap parameter? It didn't look like the usd-arnold code ever added entries to this map, for instance.

If we can make this const, that'd be great. If we need it to be non-const, could we change it to be a non-const pointer instead of a reference? That's our usual coding convention for parameters that may be modified.